components/Footer.js

```diff
@@ -1,7 +1,81 @@
 import React from "react";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='dark:bg-neutral-900 dark:text-gray-50 py-6'>
+			<div className='container p-6 mx-auto lg:space-y-8'>
+				<h2 className='text-4xl font-lexend font-bold text-center'>
+					Let&#39;s connect!
+				</h2>
+				<div className='flex flex-wrap justify-center justify-around px-80'>
+					<a
+						rel='noopener noreferrer'
+						href='https://www.instagram.com/acm_ucr/'
+						target='_blank'
+						title='Instagram'
+						className='w-12 h-12 mx-10 my-6 fill-current md:mx-12 lg:m-0 dark:text-gray-50 hover:text-purple-500'
+					>
+						<svg
```

Comment:
What is the SVG for? Can we switch over to using React Icons? Same goes for the path component